src/core/Utils.js

```diff
@@ -401,6 +401,27 @@ function findDOMElement (element) {
   }
 }
 
+/**
+* Get metadata object from a form element
+*
+* @param {Node|string} element
+* @return {Object|null}
+*/
+
+function getMetaFromForm (element) {
+  if (!element || element.tagName !== 'FORM') {
+    console.error('Metadata can only be extracted from form elements')
+    return null
+  }
+
+  var formData = new FormData(element)
+  var result = {}
+  for (var entry of formData.entries()) {
```

Comment:
Hm, good find! So, are form elements with data mostly inputs? I could manually loop through all inputs in form, setting their `name` as key and `value` as value on the `formMetaData` object.